pybot/Robot.py

```diff
@@ -213,13 +218,27 @@ def appliquer_filtre(self, chemin_fichier, nom_filtre):
         self.ecran.set_filter(chemin_fichier, nom_filtre)
 
     ### RECONNAISANCE CARTES - SESSION UTILISATEUR ###
-        
-    def detecter_carte(self):
+
+    def detecter_carte(self, seuil_minimal=0.75, seuil_arret_recherche=0.85):
         """
-            ...
+            Affiche à l' écran un cadre autour de la carte.\n
+            Retourne 'nom prénom': soit le nom et prénom associé à la carte détectée\n
+            Retourne '': si aucune carte n' est détectée ou si Robot a mal été initalisé
+
+            Paramètres:
+                - seuil_minimal (défaut: 0.75) : score minimum pour qu' une carte détectée soit considérée comme valide.
+                - seuil_arret_recherche (défaut: 0.85) : score pour qu' une carte détectée soit interprétée comme la bonne.
         """
-        return self.ecran.detect_card()
-    
+        if self.webapp is None:
+            self.message_avertissement(
+                "La fonction Robot.detecter_carte() a été appelée sans Robot.demarrer_webapp()")
+            return ""
+        users = self.ecran.detect_card()
```

Comment:
detect_card should be in its own module, it should not be a _ecran_ module dependency

Reply:
detect_card() should become connect() and return a full db user

Reply:
add the supprimer_utilisateur() method that disconnect the **connected** user and removes it from the db